worlds/minecraft/docs/en_Minecraft.md

```diff
@@ -9,7 +9,7 @@ config file.
 
 Recipes are removed from the crafting book and shuffled into the item pool. It can also optionally change which
```

Comment:
while we are at it I would also like this reworded 
```suggestion
Some recipes are locked from being able to be crafted and shuffled into the item pool. It can also optionally change which
```